### PR TITLE
fix: Handle more dry-run cases

### PR DESCRIPTION
**Pull request by hariombalhara**
## What does this PR do?
- Workflow
- Seated Events
- Webhooks

Adds Dry Run success page that we can fill with more details later - [Loom](https://www.loom.com/share/6c874180e62545d1b08080703c7fee5d?from_recorder=1&focus_title=1)
image

## Mandatory Tasks (DO NOT REMOVE)

- [x] I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [x] I have updated the developer docs in /docs if this PR makes changes that would require a [documentation change](https://cal.com/docs). If N/A, write N/A here and check the checkbox.
- [x] Not Needed -  I confirm automated tests are in place that prove my fix is effective or that my feature works.

## How should this be tested?
Try doing the booking

### CODE REVIEW

File: packages/features/bookings/lib/handleConfirmation.ts [hunk 1]
```diff
@@ -338,14 +338,15 @@ export async function handleConfirmation(args: {
       const isFirstBooking = index === 0;
 
       if (!eventTypeMetadata?.disableStandardEmails?.all?.attendee) {
-        await scheduleMandatoryReminder(
-          evtOfBooking,
+        await scheduleMandatoryReminder({
```
Comment (hariombalhara) [inline packages/features/bookings/lib/handleConfirmation.ts:341]: Moved all args to single object as I needed to add a new one

File: apps/web/modules/bookings/views/booking-dry-run-success-view.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            </div>
+            <div>
+              <p>{sampleBookingInfo.attendees[0].name}</p>
+              <p>{sampleBookingInfo.attendees[0].email}</p>
+            </div>
+          </div>
+
+          <div className="font-medium">{t("where")}</div>
+          <div className="col-span-2 mb-6">{sampleBookingInfo.location}</div>
+        </div> */}
```
Comment (PeerRich) [inline apps/web/modules/bookings/views/booking-dry-run-success-view.tsx:72]: lets actually show the test data as a result. i like it
```suggestion

        <div className="border-subtle text-default mt-8 grid grid-cols-3 border-t pt-8 text-left">
          <div className="font-medium">{t("what")}</div>
          <div className="col-span-2 mb-6">{sampleBookingInfo.title}</div>

          <div className="font-medium">{t("when")}</div>
          <div className="col-span-2 mb-6">{new Date(sampleBookingInfo.startTime).toLocaleString()}</div>

          <div className="font-medium">{t("who")}</div>
          <div className="col-span-2 mb-6">
            <div className="mb-3">
              <div>
                <span className="mr-2">{sampleBookingInfo.user.name}</span>
                <Badge variant="blue">{t("Host")}</Badge>
              </div>
              <p className="text-default">{sampleBookingInfo.user.email}</p>
            </div>
            <div>
              <p>{sampleBookingInfo.attendees[0].name}</p>
              <p>{sampleBookingInfo.attendees[0].email}</p>
            </div>
          </div>

          <div className="font-medium">{t("where")}</div>
          <div className="col-span-2 mb-6">{sampleBookingInfo.location}</div>
        </div>
```

Comment (PeerRich) [reply apps/web/modules/bookings/views/booking-dry-run-success-view.tsx:72]: would actually need the actual dryRun results here, not static

Comment (hariombalhara) [reply apps/web/modules/bookings/views/booking-dry-run-success-view.tsx:72]: Yeah, thats planned for a followup as it has other dry run fixes as well.

**Review by PeerRich — APPROVED** [review]
looks solid

**Pull request merged**